README.md

```diff
@@ -1,2 +1,55 @@
-# go-wasi-policy-template
-Template of a plain WASI policy written using Go
+> **WARNING:** this is not the recommended way to write Kubewarden
+> policies using Go. Please read TODO for more information.
```

Comment:
Are you referring to the TODO inside the code?

Reply:
It's meant to be the link to the https://deploy-preview-195--silly-bunny-8cedd0.netlify.app/writing-policies/wasi section. I've updated the README to include the final URL